File: Step5-Python/10818.py
#way1
n=int(input())
arr=list(map(int,input().split()))
print(min(arr),max(arr))

#way2
n=int(input())
arr=list(map(int,input().split()))

max=-1000000
min=1000000

for i in arr:
    if i>max:
        max=i

    if i<min:
        min=i

print(min,max)

File: Step5-C/10818.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int main()
{
	int n;
	scanf("%d", &n);

	int arr;
	int min = 1000000;
	int max = -1000000;
	for (int i = 0; i < n; i++)
	{
		scanf("%d", &arr);

		if (min > arr)
			min = arr;
		if (max < arr)
			max = arr;
	}
	printf("%d %d", min, max);

	return 0;
}